app/models/fee.rb

```diff
@@ -24,7 +24,8 @@ class Fee < ApplicationRecord
   monetize :vat_amount_cents
   monetize :total_amount_cents
 
-  FEE_TYPES = %i[charge add_on subscription credit instant_charge].freeze
+  # TODO: Deprecate add_on type in the near future
+  FEE_TYPES = %i[charge add_on subscription credit instant_charge one_off].freeze
```

Comment:
The plan is to remove add_on fee type in the future since applied_add_on feature will be deprecated. Also, probably one-off invoice will not necessary be related only to add-ons. So it seems to me reasonable to add new type here. wdyt?

Reply:
Not a blocker for me, but since for now it is still add_on related I would keep it, and maybe change it with the future removal of the applied_add_ons